trustwallet/wallet-core
# [Security]: Revert TON Mnemonic in dev branch

## Description

Revert TON 24-words specific mnemonic as we identified a security issue - possible DoS due to too strict restrictions of the TON Mnemonic words.
TON team is aware of their bad practice, that they don't follow the standard to generate private key: https://github.com/toncenter/tonweb-mnemonic/issues/13

## How to test

Run Rust, C++, iOS, Android, WASM tests
In addition, check https://github.com/trustwallet/wallet-core/compare/master...s/dev-revert-ton-mnemonic - there must not be any changes related to TON

## Types of changes

Revert `0b167710b325d299398ca82ef57ee8e630e54633` commit

## Checklist




If you're adding a new blockchain
